devops/openshift/workers-deploy.yml

```diff
@@ -124,8 +124,10 @@ parameters:
     value: superuser-username
   - name: DB_PASSWORD_KEY
     value: superuser-password
-  - name: DATABASE_NAME_KEY
-    required: true
+  - name: API_SECRET_NAME
```

Comment:
API_SECRET_NAME is not required to be added here. `WORKERS_SECRET_NAME` can be re-used.